src/Enum/Enum.php

```diff
@@ -69,6 +70,17 @@ public static function getAvailableValues()
 	}
 
 	/**
+	 * @return static[] format: const name (string) => instance (Enum)
```

Comment:
> instance (Enum)

Please use FQN name as in other PHPDoc cases.
But probably `static` should be here as well.